src/api/heartbeat.coffee

```diff
@@ -1,11 +1,27 @@
-utils = require "../utils"
-server = require "../server"
+utils = require '../utils'
+server = require '../server'
+Mediator = require('../model/mediators').Mediator
+moment = require 'moment'
 Q = require 'q'
 
 exports.getHeartbeat = ->
   try
     uptime = Q.denodeify( server.getUptime )
     result = yield uptime
+
+    mediators = yield Mediator.find().exec()
+    for mediator in mediators
+      if not result.mediators then result.mediators = {}
+
+      isUp = -> mediator._lastHeartbeat? and mediator._uptime? and
+      # have we received a heartbeat within the last minute?
+      moment().diff(mediator._lastHeartbeat, 'seconds') <= 60
+
+      if isUp mediator
```

Comment:
It seems `isUp` doesn't take a `mediator` parameter, it just closes over the `mediator` variable in the current scope.